docs/_articles/en/apex/writing.md

```diff
@@ -97,8 +101,13 @@ Declare Missing Methods quick fix can be invoked in the following ways:
 > Note: Keyboard shortcut for the Quick Fix widget is Cmd+. in macOS and Ctrl+. in Windows and Linux.
 
 ## Anonymous Apex
+It’s common to keep Apex code in your project for executing certain tasks. By default, a new project has a folder `scripts/`apex that contains an example `hello.apex` file. We recommend that you use this folder to create your anonymous Apex files using the `.apex` file extension.
```

Comment:
here anonymous is l/c. Whatever you find out, be consistent.

Reply:
Should be Ic per Apex Guide